fqd.c

```diff
@@ -158,15 +164,71 @@ static uint32_t get_my_ip(void) {
   }
   return 0;
 }
+void signal_handler(int sig, siginfo_t *info, void *ucontext) {
+	bcd_fatal("This is a fatal crash");
+	raise(sig);
+	return;
+}
 int main(int argc, char **argv) {
+  struct sigaction new_action;
+  bcd_error_t error;
+	bcd_t bcd;
+  new_action.sa_handler = signal_handler;
+  sigemptyset(&new_action.sa_mask);
+  new_action.sa_flags = SA_SIGINFO;
   nodeid = get_my_ip();
   parse_cli(argc,argv);
+  if (backtrace)
```

Comment:
Use different variable name as to not conflict with glibc-provided `backtrace(3)`.